### PR TITLE
refactor: classification and tags form

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes:

I worked on classification and tags form refactoring.

Used AntD components
Used the form and removed the local states
Aligned validation with json schema

### Classification Form
image

### Tags Form

image

#
### Type of change:

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [x] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/router/AuthenticatedAppRouter.tsx
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/router/AuthenticatedAppRouter.tsx:1]: Let's not use .component for pages: https://www.notion.so/collate-inc/Developer-Guide-for-UI-UX-61a3380cb472485096d521e46f3148bd?pvs=4

File: openmetadata-ui/src/main/resources/ui/src/utils/TagsUtils.tsx [hunk 1]
```diff
@@ -272,3 +276,32 @@ export const getTagTooltip = (fqn: string, description?: string) => (
     </div>
   </div>
 );
+
+export const getDeleteIcon = (arg: {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/utils/TagsUtils.tsx:280]: Tests for this func?

File: openmetadata-ui/src/main/resources/ui/src/utils/TagsUtils.tsx [hunk 2]
```diff
@@ -272,3 +276,32 @@ export const getTagTooltip = (fqn: string, description?: string) => (
...
+      return <CheckOutlined />;
+    }
+
+    return <Loader size="small" type="default" />;
+  }
+
+  return <DeleteIcon name="Delete" width={16} />;
+};
+
+export const getUsageCountLink = (tagFQN: string) => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/utils/TagsUtils.tsx:297]: tests for this func?

File: openmetadata-ui/src/main/resources/ui/src/components/router/AuthenticatedAppRouter.tsx
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/router/AuthenticatedAppRouter.tsx:1]: sure, I will change that.

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js [hunk 3]
```diff
@@ -11,8 +11,6 @@
  *  limitations under the License.
  */
 
-// / <reference types="cypress" />
-
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js:15]: why we are removing this line? it helps to provide suggestions in Cypress
https://docs.cypress.io/guides/tooling/IDE-integration

**Review by ShaileshParmar11 — COMMENTED** [review]
one comment

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js (hunk 3 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Tags.spec.js:15]: No worries, I will add it.

**Review by chirag-madlani — APPROVED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**